.github/pull_request_template.md

```diff
@@ -1,3 +1,3 @@
 <!--
-Hi there 👋, please check you've read the [CONTRIBUTING](../CONTRIBUTING.md) guide before submitting a PR (it's worth it, honestly).
+Hi there 👋, please check you've read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) guide before submitting a PR (it's worth it, honestly).
```

Comment:
Does not work when I'm creating a PR on https://github.com/matrix-org/matrix-hookshot/compare/main...my-awesome-branch

Also, it does not get rendered as HTML anyway, because it's a comment.

Let's not link this in our template. For people who submit a PR for the first time, GitHub will show a notice with a link.

```suggestion
Hi there 👋, please check you've read the guide in CONTRIBUTING.md before submitting a PR (it's worth it, honestly).
```

Reply:
> Does not work when I'm creating a PR on https://github.com/matrix-org/matrix-hookshot/compare/main...my-awesome-branch

I'm not versed that part of GitHub magic, but I think that is because it's not merged to main yet?

Other than that, you're not wrong.

Other repos such as synapse do:
```
<!-- Please read https://matrix-org.github.io/synapse/latest/development/contributing_guide.html before submitting your pull request -->
```
which is a bit special since their docs are some web page. However I think usually people create PRs in the browser? So copy-pasting a link seems perhaps nicer than navigating manually.